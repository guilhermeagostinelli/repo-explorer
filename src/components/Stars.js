import styled from "styled-components";

const Stars = styled.p`
  ::before {
    content: "‚≠ê";
    margin-right: 3px;
  }
`;

export default Stars;
