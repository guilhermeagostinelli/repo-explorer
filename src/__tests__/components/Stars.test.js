import React from "react";
import { create } from "react-test-renderer";
import Stars from "../../components/Stars";
import "jest-styled-components";

describe("<Stars />", () => {
  it("should match the snapshot", () => {
    const stars = create(<Stars />);
    expect(stars.toJSON()).toMatchSnapshot();
  });
});
