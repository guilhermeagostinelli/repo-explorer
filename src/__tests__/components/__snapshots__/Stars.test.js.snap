// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stars /> should match the snapshot 1`] = `
.c0::before {
  content: "‚≠ê";
  margin-right: 3px;
}

<p
  className="c0"
/>
`;
